pkgs/tools/networking/picosnitch/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, psutil, dbus-python, requests }:
+
+buildPythonPackage rec {
+  pname = "picosnitch";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e56aef62f0f9a48a346f02a96362b0e655cc754ebacaf58cb1632e602463d365";
+  };
+
+  propagatedBuildInputs = [ psutil dbus-python requests ];
+
+  pythonImportsCheck = [ "picosnitch" ];
+
+  meta = with lib; {
+    description = "An extremely simple, reliable, and lightweight program for linux to help protect your privacy";
+    homepage = "https://github.com/elesiuta/picosnitch";
+    changelog = "https://github.com/elesiuta/picosnitch/releases";
+    license = licenses.gpl3;
```

License seems to be [`GPLv3+`](https://github.com/elesiuta/picosnitch/blob/master/setup.py#L27).

```suggestion
    license = licenses.gpl3Plus;
```